public/pages/CreateMonitor/containers/CreateMonitor/CreateMonitor.js

```diff
@@ -242,6 +254,8 @@ export default class CreateMonitor extends Component {
       monitor = { ...monitor, ...triggers };
     }
 
+    if (monitor.monitor_type === MONITOR_TYPE.COMPOSITE_LEVEL) delete monitor.monitor_type;
```

Comment:
Can you add a comment in the code explaining why we need to delete this for composite monitor?

Reply:
This is removed, not needed since we are using workflow API now.